collections/_services/chefs.md

```diff
@@ -8,24 +8,21 @@ version: '<span class="badge badge-pill badge-warning">Beta</span>'
 order: 3
 description: >-
    At last! A user-friendly, hosted service for teams to create and publish their own web forms.   Complex form designs, custom access control, secure and highly available - made simple and maintained by the Common Services team and available for your next web form project.
-intro: >-
-  We've made things easy for you!
-onboard: ['Visit https://chefs.pathfinder.gov.bc.ca/app/ to start building your first form.']
+onboard: ['Hosted Service', 'Self-hosted']
+onboardDescription: We provide a <em>Hosted Service</em> as well as documented <em>open-source code</em> for hosting your own form service.
+featureList: |
+  - **User-friendly form designer** - Drag and Drop form components onto your form; Export and import your form design templates
+  - **Form data review** - View and export form submission data
+  - **Access controls** - Choose who can access your forms, secured with IDIR (and more options coming soon)
+  - **Team management** - Configure role-based access for your team to manage forms and review submissions
+  - **Notifications** - Receive Emails when your forms are submitted; Provide form respondents with email receipts
+  - **Configurable form actions** - Configure form actions and custom components like the <a href="https://www.orgbook.gov.bc.ca/en/home" target="_blank">BC Orgbook</a>
```

Comment:
This could probably just go as the content of the card and the place that uses it knows to use the {{ content }} in what it defines as the featureList. 
The intent with the content is that if the card were to be used in another spot where it sums up the capabilities it would just print out the content. But that hasn't ben actualized anywhere (and might not be) so it's kind of fuzzy future-guessing at this point.

But if you look at CHES and CDOGS service cards you see the features as the contents whereas other details are metadata in the head matter.

Doesn't matter too much at this point so up to you. But would be consistent with the other 2 services if that works out for the layout.

Reply:
Yeah. i think youre right. i like that idea.